### DISCUSSION TITLE
Layer "sub" labels?

### DISCUSSION THREAD

**Discussion by Cashe93** [Riverscapes/RiverscapesXML#170] [Q&A] [answered]
How do we add labels to layers that carry over with our symbology? For instance, our slope raster is in degrees and the layer appears with the unit 'Degree °' in ArcMap.

![image](https://user-images.githubusercontent.com/49880208/126911217-932c1591-51aa-44e1-9d2d-675240896eb7.png)

Could someone refresh me on how we do this in Arcmap? Also, is there a way to do this in QGIS as well? Any help would be great. Thanks!

**Comment by philipbaileynar** [comment]
You edit the layer properties and then edit the bold row that is marked "Heading".

These labels are defined in the layer or QML files, not in the business logic.

<img width="683" alt="Screen Shot 2021-07-26 at 9 18 05 PM" src="https://user-images.githubusercontent.com/8645379/127094721-0b2a94ce-4cc8-462f-83e2-2c079e1f7977.png">

**Comment by joewheaton** [comment, marked as answer]
In ArcMap, you simply click on this label in TOC and edit it.  
![image](https://user-images.githubusercontent.com/9723905/127185894-e3418476-ecc8-42bf-81ff-5a1a54fd5170.png)

I have not seen a place in QGIS to do this nor in the QML. So I am not sure if it exists or not.
